src/Palette/PaletteManager.h

```diff
@@ -2,6 +2,7 @@
 #include "impl_format.h"
 
 #include "CharPaletteHandle.h"
+#include <sstream>
```

Comment:
Not a big deal at all but this should probably be included in the cpp file rather than here.